package.json

```diff
@@ -33,9 +33,11 @@
     "@pika/plugin-build-node": "^0.9.1",
     "@pika/plugin-build-web": "^0.9.1",
     "@pika/plugin-ts-standard-pkg": "^0.9.1",
+    "@types/express": "^4.17.13",
     "@types/fetch-mock": "^7.3.1",
     "@types/jest": "^27.0.0",
     "@types/node": "^14.0.1",
+    "express": "^4.18.1",
```

Comment:
it would be great if we could update this pull request to use `createServer` from the built-in `http` module instead of depending on `express`. If anyone would be up to help with that I'd appreciate it.

Reply:
Done. Had to create another PR though: https://github.com/octokit/plugin-throttling.js/pull/539